.circleci/config.yml

```diff
@@ -155,14 +155,17 @@ commands:
       - run:
           name: Install asdf
           command: |
+            git config advice.detachedHead false
             if [ ! -d $HOME/.asdf ]; then
               git clone https://github.com/asdf-vm/asdf.git $HOME/.asdf --branch << parameters.asdf_version >>
               cd $HOME/.asdf
             else
               cd $HOME/.asdf
               git fetch origin
-              git pull
+              git fetch --all
```

Comment:
would `git fetch --tags` be enough here?

Reply:
done